### DISCUSSION TITLE
How to handle bulk imports in Shopify with webhook connection.

### DISCUSSION THREAD

**Discussion by sebszocinski** [statamic-rad-pack/shopify#141] [Help]
We're working on a store in development and processed an import of 300 products which triggered 300 webhooks to our Statamic install and caused the web server to fall-over.

Running `php please stache:clear` and `cache:clear` seemed to bring it back to life, so my guess is the stache became corrupt?

Any ideas or suggestions on how to avoid this? or is this a potential bug/issue?

**Comment by ryanmitchell** [comment]
Are you running a queue (eg redis) for the webhook processing jobs?
